src/main/java/ome/formats/importer/ImportConfig.java

```diff
@@ -648,11 +648,17 @@ protected <T> void prompt(Value<T> value, String prompt, boolean hide) {
      *  if can't log in request needed information
      */
     public void requestFromUser() {
-        if (!canLogin()) {
-            loadAll();
-            prompt(hostname, " Enter server name: ", false);
-            prompt(username, " Enter user name: ", false);
-            prompt(password, " Enter password: ", true);
+        if (hostname.empty()) {
+            hostname.set(System.console().readLine("Hostname: "));
```

Comment:
To unify with text display when doing ``omero import XX``, this could be named ``"Server: "``

Reply:
@chris-allan was faster than me but I had the exact same hesitations.

For completeness the logic of the Python CLI is defined in https://github.com/ome/omero-py/blob/ccfdadd131bf0f1bb4c4ac902a5c2aa8cbc712f5/src/omero/plugins/sessions.py#L979-L988 for the server prompt and https://github.com/ome/omero-py/blob/ccfdadd131bf0f1bb4c4ac902a5c2aa8cbc712f5/src/omero/plugins/sessions.py#L966-L977 for the input parsing.

It is worth noting the current logic goes as far as supporting of a full `<user>@<host>:<port>` input. But the prompt message definitely suggests setting the port as `<host>:<port>` while in this case a non default port must be supplied via a different option

Reply:
Agree it is a tricky one.